src/02_spec/07_modules/cdm/systemif.rst

```diff
@@ -0,0 +1,26 @@
+System Interface
+----------------
+
+The following signals are required from the OR1K CPU core in order to interface to the CDM-OR1K module: 
+
++-----------------+----------------------+---------------+------------------------------------------------+
+| Signal          | Width                | Direction     | Description                                    |
++=================+======================+===============+================================================+
+| ``du_stall_i``  | 1                    | CDM->CPU      | Logic '1' causes CPU to stall                  |
++-----------------+----------------------+---------------+------------------------------------------------+
+| ``du_stall_o``  | 1                    | CPU->CDM      | Indicates CPU has reached breakpoint condition |
++-----------------+----------------------+---------------+------------------------------------------------+
+| ``du_stb_i``    | 1                    | CDM->CPU      | Access to the core debug interface             |
++-----------------+----------------------+---------------+------------------------------------------------+
+| ``du_ack_o``    | 1                    | CPU->CDM      | Complete access to the core                    |
++-----------------+----------------------+---------------+------------------------------------------------+
+| ``du_adr_i``    | 16                   | CDM->CPU      | Address of CPU register to be read or written  |
++-----------------+----------------------+---------------+------------------------------------------------+
+| ``du_we_i``     | 1                    | CDM->CPU      | Write cycle when true, read cycle when false   |
++-----------------+----------------------+---------------+------------------------------------------------+
+| ``du_dat_i``    | OPTION_OPERAND_WIDTH | CDM->CPU      | Write data                                     |
```

Comment:
OPTION_OPERAND_WIDTH is defined to be what? The implementation your're describing here works only with 32 bit CPUs, so you can leave that fixed.